framework/lib/components/step-stack/step-stack.tsx

```diff
@@ -3,30 +3,56 @@ import { useToken } from '@chakra-ui/system'
 import { StepStackProps } from './types'
 import { getChildrenWithProps } from '../../utils'
 import { HStack, Stack } from '../stack'
-import { Center } from '../center'
 import { Divider } from '../divider'
 import { Label } from '../typography'
+import { Box } from '../box'
 /**
- * Util wraper for creating ordered vertical layouts by stack
- * @see {@link https://northlight.dev/reference/step-stack}
- * @example
- * (?
- * <StepStack maxW="sm" spacing="4" rowHeight="10">
-    { Array.from({length: 5}, (_, i) => i).map((i) => <Input key={ i } />) }
-</StepStack>
- * ?)
-<br />
-It takes all the props that a normal stack takes, with the addition of a
- rowHeight prop, which is needed to get the correct height for the grey
+  Util wrapper for creating ordered vertical layouts by stack
+  @see {@link https://northlight.dev/reference/step-stack}
+  @example
+  You can adjust the vertical positioning of the step
+  circles using the <Code>stepCircleAlignment</Code> prop.
+  This prop accept any values that can be used with the alignItems CSS property.
+  If no <Code>stepCircleAlignment</Code> value
+  is passed then the default value of <Code>center</Code> will be used
+  for the <Code>stepCircleAlignment</Code> prop.
+  <br /><br />
```

Comment:
I'd prob put these comments about the use of props under the step-stack/types.ts file so it renders like this
![Image 2023-09-29 at 10 17](https://github.com/mediatool/northlight/assets/90923099/c0439764-11fb-482e-ae2f-b710cf1c3143). Its a nit picky thing tho and it looks good otherwise